ls1intum/Artemis
# [Bugfix] Fix Problem Statement in Programming Assessment Instructions

### Checklist

### Motivation and Context


Previously the problem statement showed that all tests failed even though the test did actually passed (seen in the result details/feedback)

### Description

- Added new input to handover the programming participation during the manual assessment

### Steps for Testing


1. Log in to Artemis
2. Navigate to Course Administration
3. Assess a Programming Exercise
4. Check that the passed tests are correctly displayed in the problem statement of the assessment instructions

### Test Coverage



Nothing changed.

### Screenshots


<img width="1668" alt="image" src="https://user-images.githubusercontent.com/41108487/106528992-02617b80-64ea-11eb-98d4-23f9d92cf654.png">
